.github/workflows/ci.yml

```diff
@@ -75,8 +92,6 @@ jobs:
     strategy:
```

Comment:
we should make this job require rustfmt as a pre-requisite, otherwise CI will continue even if the formatter checks fail.

Reply:
Specifically the "needs" section for `cargo-verifications` should include `rustfmt`